web/src/compose_reply.js

```diff
@@ -120,12 +124,19 @@ export function respond_to_message(opts) {
 
 export function reply_with_mention(opts) {
     respond_to_message(opts);
+    assert(message_lists.current !== undefined);
     const message = message_lists.current.selected_message();
     const mention = people.get_mention_syntax(message.sender_full_name, message.sender_id);
     compose_ui.insert_syntax_and_focus(mention);
 }
 
 export function quote_and_reply(opts) {
+    if (message_lists.current === undefined) {
+        blueslip.error(
+            "message_lists.current expected to be defined for current hash." + window.location.hash,
+        );
+    }
```

Comment:
Hmm, why isn't this one an assert? I think this gets called from the same scenarios as `reply_with_mention`.

Reply:
Yeah, I am not sure why I did that.